### PR TITLE
About website

### PR DESCRIPTION
**Pull request by adaortizcarbonell**
A little revamping of the About paragraph

### CODE REVIEW

File: content/about/employees/ada/index.md [hunk 1]
```diff
@@ -10,16 +10,12 @@ anchor: ada
 services:
   - datascience
 summary: |
-    Astrophysicist researcher turned into data scientist and having fun with it. After 20 years in academia I decided to take on a new challenge and apply my skills in the industry. 
+    Ada is a senior researcher with 20 years of academic experience in the field of Astrophysics during which she taught, mentored students, organized several international conferences and led her own research projects. She is engaged in scientific communication & outreach, and has published two books, delivered public lectures and appeared in the media (newspaper & radio) explaining solar physics to a general audience. She decided to take on a new challenge and apply her skills in the industry. Ada enjoys analizing complex problems and extracting valuable information out of data.
```
Comment (eigilhs) [inline content/about/employees/ada/index.md:13]:
```suggestion
    Ada is a senior researcher with 20 years of academic experience in the field of Astrophysics during which she taught, mentored students, organized several international conferences and led her own research projects. She is engaged in scientific communication and outreach, and has published two books, delivered public lectures and appeared in the media (newspaper and radio) explaining solar physics to a general audience. She decided to take on a new challenge and apply her skills in the industry. Ada enjoys analyzing complex problems and extracting valuable information out of data.
```
Or maybe drop the parenthesis completely to make it easier?

File: content/about/employees/ada/index.md [hunk 2]
```diff
@@ -10,16 +10,12 @@ anchor: ada
...
-feature recognition algorithms and visualization tools for large volumes of data. Such skills can also be applied to complex problems in the private industry.
+which she obtained in 2003. She has 20 years of experience in academia doing research in the field of Solar Physics. This background provided her with experience in handling large datasets from different spacecrafts and ground-based observatories, scientific programming, image
+analysis, time-series analysis, analytical modelling, signal processing, as well as the development of feature recognition algorithms and visualization tools for large volumes of data. Such skills can also be applied to complex problems in the private industry.
 
 Ada has an analytical mind, which together with thoroughness and perseverance provides her with the
 ability to solve problems. Her interests expand beyond the purely scientific domain into education,
-scientific communication & outreach, logistics and organization. Her peers regard her as an excellent communicator and an
-all-terrain asset.
+scientific communication & outreach, and organization. Her peers regard her as an excellent communicator and an all-terrain asset.
```
Comment (eigilhs) [inline content/about/employees/ada/index.md:21]:
```suggestion
scientific communication and outreach, and organization. Her peers regard her as an excellent communicator and an all-terrain asset.
```

**Review by eigilhs — COMMENTED** [review]
A typo and suggest changing ampersands to "and"s.

File: content/about/employees/ada/index.md (hunk 1 above)
Comment (adaortizcarbonell) [reply content/about/employees/ada/index.md]: takk!

File: content/about/employees/ada/index.md (hunk 2 above)
Comment (adaortizcarbonell) [reply content/about/employees/ada/index.md]: takk!

**Review by eigilhs — APPROVED** [review]

**Pull request merged**